jekyll/_cci2/server-3-install-build-services.adoc

```diff
@@ -678,31 +678,31 @@ gcloud projects add-iam-policy-binding <YOUR_PROJECT_ID> --member serviceAccount
 
 . *Get JSON Key File*
 +
-After running the following command, you should have a file named `circleci-server-vm-keyfile` in your local working directory. You will need this when you configure your server installation.
+After running the following command, you should see a file named `circleci-server-vm-keyfile` in your local working directory. You will need this file when you configure your server installation:
 +
 ```shell
 gcloud iam service-accounts keys create circleci-server-vm-keyfile --iam-account <YOUR_SERVICE_ACCOUNT_EMAIL>
 ```
 
 . *Configure Server*
 +
-Configure VM Service through the KOTS admin console. Details of the available configuration options can be found in the https://circleci.com/docs/2.0/server-3-operator-vm-service[VM Service] guide.
+Configure VM Service through the KOTS Admin Console. Details of the available configuration options can be found in the https://circleci.com/docs/2.0/server-3-operator-vm-service[VM Service] guide.
 +
 Once you have configured the fields, *save your config* and deploy your updated application.
 
 endif::env-aws[]
 
 ==== VM Service Validation
 
-Once you have configured and deployed CircleCI server you should validate that VM Service is operational. You can rerun the reality checker project within your CircleCI installation and you should see the VM Service Jobs complete with green. At this point all tests should pass with green.
+Once you have configured and deployed CircleCI server, you should validate that VM Service is operational. You can rerun the realitychecker project within your CircleCI installation and you should see the VM Service Jobs complete with green. At this point, all tests should pass with green.
```

Comment:
"with green" sounds really weird to me. Pass is enough I think. What do you think @catherinepope?

Reply:
I agree!